modules/openapi-generator/src/main/resources/nodejs-express-server/controllers/Controller.mustache

```diff
@@ -2,138 +2,117 @@ const fs = require('fs');
 const path = require('path');
 const config = require('../config');
 const logger = require('../logger');
+const camelCase = require('camelcase');
 
 class Controller {
-  static sendResponse(response, payload) {
-    /**
-     * The default response-code is 200. We want to allow to change that. in That case,
-     * payload will be an object consisting of a code and a payload. If not customized
-     * send 200 and the payload as received in this method.
-     */
-    response.status(payload.code || 200);
-    const responsePayload = payload.payload !== undefined ? payload.payload : payload;
-    if (responsePayload instanceof Object) {
-      response.json(responsePayload);
-    } else {
-      response.end(responsePayload);
-    }
-  }
+static sendResponse(response, payload) {
+/**
+* The default response-code is 200. We want to allow to change that. in That case,
+* payload will be an object consisting of a code and a payload. If not customized
+* send 200 and the payload as received in this method.
+*/
+response.status(payload.code || 200);
+const responsePayload = payload.payload !== undefined ? payload.payload : payload;
+if (responsePayload instanceof Object) {
+response.json(responsePayload);
+} else {
+response.end(responsePayload);
+}
+}
+
+static sendError(response, error) {
+response.status(error.code || 500);
+if (error.error instanceof Object) {
+response.json(error.error);
+} else {
+response.end(error.error || error.message);
+}
+}
```

Comment:
@YishTish thanks for the PR. I notice all the indentations are gone. Is that intentional?